packages/mui-joy/src/Drawer/Drawer.tsx

```diff
@@ -191,6 +192,7 @@ const Drawer = React.forwardRef(function Drawer(inProps, ref) {
     color,
     variant,
     size,
+    scrollLockContainer,
```

Comment:
No need to override this in `ownerState` as it does not have a default value and is already spread by `props`.
```suggestion
```